src/annotator/integrations/html-metadata.js

```diff
@@ -183,27 +151,31 @@ export class HTMLMetadata {
     return tags;
   }
 
-  _getTitle() {
-    if (this.metadata.highwire.title) {
-      this.metadata.title = this.metadata.highwire.title[0];
-    } else if (this.metadata.eprints.title) {
-      this.metadata.title = this.metadata.eprints.title[0];
-    } else if (this.metadata.prism.title) {
-      this.metadata.title = this.metadata.prism.title[0];
-    } else if (this.metadata.facebook.title) {
-      this.metadata.title = this.metadata.facebook.title[0];
-    } else if (this.metadata.twitter.title) {
-      this.metadata.title = this.metadata.twitter.title[0];
-    } else if (this.metadata.dc.title) {
-      this.metadata.title = this.metadata.dc.title[0];
+  /** @param {HTMLDocumentMetadata} metadata */
+  _getTitle(metadata) {
+    if (metadata.highwire.title) {
```

Comment:
At some point it might be nice to codify the strategy here with some comments. Yeah, it's certainly readable—look for the highwire title, first, etc., but what's lost here (and maybe lost forever anyway if no one knows it) is the intent behind this order of prioritization. Like, why did we decide that `highwire` is the most powerful source of title-truth? Just musing...

_Updated_: Some of the added title-related tests help to shed at least a little light here by explaining, at least, the order of priority.

Reply:
I don't know the rationale behind the original priority. It might have been pretty arbitrary and it might not be optimal. For the moment I've just made sure the current behavior is at least tested.